src/pages/docs/faq.md

```diff
@@ -293,6 +293,20 @@ Long axis labels can be problematic. There are several ways to address the issue
 </div>
 ```
 
+### My axis labels are showing very small numbers. How do I fix this?
+
+When a dataset is provided with all values on an axis being `null` or `0`, Victory will set the axis domain to be between zero and a very small number, to avoid any errors.
```

Comment:
This edge case is hit if all the values in a given dimension are _the same_. This usually occurs when a chart contains only one point like `[{ x: 1, y: 1 }]`, but can also occur with multiple points as you show below. Victory will create a two point domain by padding the single data value with a very small number.